fbw-a380x/src/systems/systems-host/systems/FlightWarningSystem/FwsCore.ts

```diff
@@ -460,15 +471,14 @@ export class FwsCore {
 
   public readonly fms2Fault = Subject.create(false);
 
-  public readonly autoPilotDisengagedInstantPulse = new NXLogicPulseNode(false);
-
-  public readonly autoPilotDisengagedDelay = new NXLogicTriggeredMonostableNode(0.1, true);
+  /** If multiple AP discs are triggered between FWS cycles, memorize the amount */
+  public autoPilotInstinctiveDiscCountSinceLastFwsCycle = 0;
 
-  public readonly autoPilotDisengagedDelayedPulse = new NXLogicPulseNode(false);
+  public readonly autoPilotDisengagedInstantPulse = new NXLogicPulseNode(false);
 
-  public readonly autoPilotInstinctiveDiscPressedInLast1p8Sec = new NXLogicTriggeredMonostableNode(1.9, false);
+  public readonly autoPilotInstinctiveDiscPressedInLast1p8Sec = new NXLogicTriggeredMonostableNode(1.9, true);
```

Comment:
The variable name doesn't match the 1.9s you give it

Reply:
Thanks for the review. The references say 1.8s, but due to our audio files I had to raise it to 1.9s to that the triple click can finish before the loud CRC. I added a comment, we could maybe touch that again when we have proper triple click logic